org.ryujinx.Ryujinx.json

```
@@ -0,0 +1,1291 @@
+{
+    "app-id": "org.ryujinx.Ryujinx",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "19.08",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions": ["org.freedesktop.Sdk.Extension.dotnet"],
+    "command": "start",
+    "finish-args" : [
+        "--share=ipc",
+        "--share=network",
+        "--device=dri",
+        "--socket=x11",
+        "--filesystem=host"
+    ],
+    "modules": [
+        {
+            "name": "start",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -d /app/bin",
+                "install -Dm555 start /app/bin"
+            ],
+            "sources": [
+                {
+                    "type": "script",
+                    "dest-filename": "start",
+                    "commands": [
+                        "DOTNET_ROOT=/app/dotnet/lib/ /app/ryujinx/Ryujinx"
+                    ]
+                }
+            ]
+        },
+        {
+            "name": "dotnet",
+            "buildsystem": "simple",
+            "build-commands": [
+                "/usr/lib/sdk/dotnet/install.sh"
+            ]
+        },
+        {
+            "name": "ryujinx",
+            "buildsystem": "simple",
+            "build-commands": [
+                "mkdir /app/ryujinx",
+                ". /usr/lib/sdk/dotnet/enable.sh; dotnet publish -r linux-x64 -c Release --source /run/build/ryujinx/nuget-sources --output /app/ryujinx" ,
+                "install -D -m0644 Ryujinx/Ui/assets/Icon.png /app/share/app-info/icons/flatpak/128x128/org.ryujinx.Ryujinx.png",
+                "ln -s /app/ryujinx/Ryujinx /app/bin"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/Ryujinx/Ryujinx",
+                    "commit": "1bb7fdac"
```

It should use full commit hash.
```suggestion
                    "commit": "e7d09a8080f6a99b099f2b36f8a26a0c6a89cef7"
```